rllib/utils/metrics/metrics_logger.py

```diff
@@ -283,21 +284,130 @@ def _map(path, stat_or_value):
 
         tree.map_structure_with_path(_map, stats_dict)
 
-    def log_n_dicts(
+    def merge_and_log_n_dicts(
         self,
-        stats_dicts,
+        stats_dicts: List[Dict[str, Any]],
```

Comment:
This is already committed in the other PR today?

Reply:
Ah, yes, very sorry. I had to make sure the other PR (MetricsLogger `log_n_dicts()` fix) is also in here otherwise, the Atari benchmark would have failed.

Thanks for pointing this out.